crates/turborepo-lib/src/task_graph/visitor.rs

```diff
@@ -381,6 +384,35 @@ impl<'a> Visitor<'a> {
 
         let global_hash_summary = GlobalHashSummary::try_from(global_hash_inputs)?;
 
+        // output any warnings that we collected while running tasks
+        if let Ok(warnings) = self.warnings.lock() {
+            if !warnings.is_empty() {
+                println!();
+                warn!("finished with warnings");
+                println!();
```

Comment:
I think we want to print all of this to `stderr`
```suggestion
                eprintln!();
                warn!("finished with warnings");
                eprintln!();
```

Reply:
Good call